{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -44,14 +44,14 @@ jobs:
     action: gh:neuro-actions/remote_debug@master
     args:
       image: $[[ images.myimage.ref ]]
-      volumes_data_ref: $[[ volumes.data.remote ]]
-      volumes_code_ref: $[[ volumes.code.remote ]]
-      volumes_config_ref: $[[ volumes.config.remote ]]
-      volumes_results_ref: $[[ volumes.results.remote ]]h
+      volumes_data_remote: $[[ volumes.data.remote ]]
```

Comment:
Do not forget to update also [this](https://github.com/neuro-actions/remote_debug/blob/master/.neuro/live.yaml) guy.